apps/tests/aws-runtime/test/tester.test.ts

```diff
@@ -378,3 +378,32 @@ test("test service context", async () => {
     serviceUrl: url,
   });
 });
+
+test("index.search", async () => {
+  const serviceClient = new ServiceClient<typeof TestService>({
+    serviceUrl: url,
+  });
+
+  await serviceClient.indexBlog({
+    blogId: "blog-id-1",
+    title: "fluffy pillows",
+    content: "i like fluffy pillows, they are super comfy",
+  });
+  await expect(
+    await serviceClient.searchBlog({
+      query: "super",
+    })
```

Comment:
Does this return a double promise? If not you should pass the promise to `await expect(promise).resolves` not the awaited value.

Reply:
I am not sure if `resolved` works correctly on a non-promise, but also you wouldn't get a nice error if the promise does reject.

Will check against the resolved value, give a clean error if rejects. (ex: expected to resolve, but rejected)

```ts
await expect(serviceClient.searchBlog({
   query: "super",
})).resolved.toEqual()
```

Will just throw if rejects.

```ts
expect(await serviceClient.searchBlog({
   query: "super",
})).toEqual()
```